prow/cmd/deck/BUILD

```diff
@@ -1,5 +1,29 @@
 package(default_visibility = ["//visibility:public"])
 
+load("@io_bazel_rules_docker//container:container.bzl", "container_image")
+
+container_image(
+    name = "asset-base",
+    base = "@alpine-base//image",
+    # With paths relative to the current directory.
+    data_path = ".",
+    # Add files under into the root directory.
+    directory = "/",
+    # Everything under static/
+    files = glob(["static/**/*"]),
+)
+
+load("@io_bazel_rules_docker//go:image.bzl", "go_image")
+
+go_image(
+    name = "image",
+    base = ":asset-base",
+    embed = [":go_default_library"],
+    importpath = "k8s.io/test-infra/prow/cmd/deck",
+    pure = "on",
+    race = "off",
```

Comment:
Can we enable race detection on all components or does it need to be disabled for some reason?

Reply:
race and pure modes are incompatible. this only affects the binaries, anyway.